lib/Mojolicious/Guides/Tutorial.pod

```diff
@@ -422,8 +422,8 @@ stash all the time.
 
 =head2 Restrictive placeholders
 
-The easiest way to make placeholders more restrictive are alternatives, you
-just make a list of possible values.
+The easiest way to make placeholders more restrictive is to use
+alternatives: you just make a list of possible values.
```

Comment:
I feel a colon is more appropriate, as the second clause is not only related, but expounds the first clause. And yes, semicolons can be used to fix some of the comma splices, but I think in technical documentation that would feel a bit clunky, compared to concise, simple sentences.